package.json

```diff
@@ -269,7 +269,7 @@
     }
   ],
   "engines": {
-    "vscode": "^1.23.0"
+    "vscode": "^1.26.1"
```

Comment:
Note that this will mean that users will need to install at least 1.26.1 in order to install C# for VS Code. Is that intentional? Does this API only appear in 1.26.1? Or, is it in an earlier version (e.g. 1.26.0).